components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue

```diff
@@ -40,9 +40,14 @@
         <span
           :class="{
             'has-text-danger': props.row.status === OfferStatusType.WITHDRAWN,
-            'has-text-success': props.row.status === OfferStatusType.ACTIVE,
+            'has-text-success':
+              props.row.status === OfferStatusType.ACTIVE &&
+              props.row.expiration >= currentBlock,
+            'has-text-warning':
+              props.row.status === OfferStatusType.ACTIVE &&
+              props.row.expiration < currentBlock,
```

Comment:
I guess it could be a function  `status === OfferStatusType.ACTIVE && expiration < currentBlock` , because I have seen twice in this file

Reply:
hmm not sure I'm understanding your concern let me know which line you refering

Reply:
<img width="608" alt="image" src="https://user-images.githubusercontent.com/31397967/213182448-0e9bc7d5-9fa3-4485-9b42-a301719d6e3f.png">


<img width="706" alt="image" src="https://user-images.githubusercontent.com/31397967/213182416-ffeae3c0-d978-4efd-a774-ecb3133bccd1.png">

Reply:
that's what I thought but `status` !== `props.row.status`? do you have any idea how to refactor this? 
I can create a new `formatOfferTextColor` method if you want it

Reply:
Sorry, I have not made it clear. I mean:  isInavtiveOffer or isInvalidOffer
```
 function isInavtiveOffer(status, expiration): boolean { 
   return status === OfferStatusType.ACTIVE && expiration < this.currentBlock
}
```